geoipdb.go

```diff
@@ -111,15 +125,21 @@ func NewHandler(overrides *mgo.Collection, timeout time.Duration) (Handler, erro
 // an ASN identification
 // and the corresponding description.
 func (h Handler) LibGeoipLookup(ip string) (string, string) {
-	if ip == "" {
+	var name string
+	ip_, is4 := iputils.ParseIP(ip)
```

Comment:
why are you using `ip_` ? I think underscore might confuse people thinking there is an ignored second return argument.